recipes/seqchromloader/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "seqchromloader" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 91efbfad647c3abc3a64880357be3f3b9de5363628b9d6adcf2b6197dccfb647
+
+build:
+  number: 0
+  noarch: generic
```

Comment:
Maybe this should be `noarch: python`? [Relevant docs](https://bioconda.github.io/contributor/guidelines.html#noarch-packages).

Reply:
I did some search on this, on [conda website](https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#architecture-independent-packages) it said 
> For pure Python packages that can run on any Python version, you can use the noarch: python value instead

Mine doesn't work with `any` Python version so that's why I put generic there, but maybe I misunderstood this sentence?

Reply:
Following your resolved comments below, I think I should follow that skip option to specify python 3.7/8/9, what I want is not to let it be able to run any python version

Reply:
I saw the link you posted in the resolved comments, seems it should be `noarch: python`, then pin the python version in requirements, thanks!